docs/import-data.md

```diff
@@ -130,6 +131,39 @@ This is a util that can help generate the import json data needed by the import
 4. Use the generated data to feed into [import] command (/import.md) to generate kick off the import.
 
 
+## Bitbucket Cloud
+*Please note that this tool uses Bitbucket cloud API version 2.0*
+1. Set the Username and Password credentials for your Bitbucket Cloud as environment variables:
+```
+export BITBUCKET_CLOUD_USERNAME=your_bitbucket_cloud_username
+export BITBUCKET_CLOUD_PASSWORD=your_bitbucket_cloud_password
+```
+2. You will need to have the organizations data in json as an input to this command to help map Snyk organization IDs and Integration Ids that must be used during import against individual targets to be imported. The following format is required:
+  ```
+  {
+    "orgData": [
+      {
+        "name": "<workspace_name_in_bitbucket_cloud_used_to_list_repos>",
+        "orgId": "<snyk_org_id>",
+        "integrations": {
+          "bitbucket-cloud": "<snyk_org_integration_id>",
+        },
+      },
+      {...}
+    ]
+  }
+  ```
+  Note: the "name" of the Bitbucket cloud workspace is required in order to list all repos belonging to that workspace via Bibucket cloud API, the Snyk specific data accompanying that workspace name will be used as the information to generate import data assuming all repos in that workspace will be imported into a given Snyk organization. This is opinionated! If you want to customize this please manually craft the import data described in [import.md](/docs/import.md)
+  - Bitbucket Cloud workspaces can be listed using the [/workspaces API](https://bitbucket.org/api/2.0/workspaces)
+  - Integrations can be listed via [Snyk Integrations API](https://snyk.docs.apiary.io/#reference/integrations/integrations/list)
+  - All organization IDs can be found by listing all organizations a group admin belongs to via [Snyk Organizations API](https://snyk.docs.apiary.io/#reference/groups/list-all-organizations-in-a-group/list-all-organizations-in-a-group)
+
+3. Run the command to generate import data:
```

Comment:
why is this in this PR? The PR says it is about `orgs:data` not about `import:data`

Reply:
Sorry, leftover from the import:data command.
Removed changes, will issue a new PR with this.